scripts/globals/caskets.lua

```diff
@@ -436,10 +450,22 @@ function getDrops(npc, dropType, zoneId)
         local itemCount    = 1
         local randomTable  = {1,4,1,3,1,1,2,1,3,1,2,1}
 
+        if zoneId == 172 or zoneId == 173 or zoneId == 190 or
+           zoneId == 191 or zoneId == 193 or zoneId == 194 or
+           zoneId == 196 or zoneId == 198 then
```

Comment:
And with a Set already defined, the same check can be used here too: `if casketInfo.splitZones[zoneId] then`

Reply:
sounds legit, ill get on that